src/main/java/frc/robot/subsystems/NewVision/NewVisionSubsystem.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.NewVision;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.photonvision.EstimatedRobotPose;
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonPoseEstimator;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Vision;
+
+public class NewVisionSubsystem extends SubsystemBase {
+  /** Creates a new NewVisionSubsystem. */
+  AprilTagFieldLayout fieldLayout;
```

Comment:
this looks like the old vision subsystem, should replace with an instance of `NewVisionIO` and one of `NewVisionIOInputs`. Then add a method to call updateinputs.